src/gluonts/distribution/distribution_output.py

```diff
@@ -127,16 +130,19 @@ def distribution(
         ----------
         distr_args
             Constructor arguments for the underlying Distribution type.
+        loc
+            Optional tensor, of the same shape as the
+            batch_shape+event_shape of the resulting distribution.
         scale
             Optional tensor, of the same shape as the
             batch_shape+event_shape of the resulting distribution.
         """
-        if scale is None:
+        if loc is None and scale is None:
             return self.distr_cls(*distr_args)
         else:
             distr = self.distr_cls(*distr_args)
             return TransformedDistribution(
-                distr, [AffineTransformation(scale=scale)]
+                distr, [AffineTransformation(loc=loc, scale=scale)]
```

Comment:
`loc` is actually never passed in any of our models, right? See [DeepAR](https://github.com/awslabs/gluon-ts/blob/master/src/gluonts/model/deepar/_network.py#L338) here for example.

Reply:
It's not indeed, and will default to `None` so long as it's not used

Reply:
To expand on this a bit: the reason for having `loc` alongside `scale` is to allow using a more general scaling, like min-max scaling or normalizing data (in the true sense of the word) instead of just dividing by some scaling factor. And if this is doable in a minimally invasive, 100% backwards compatible manner why not